lib/projects/__tests__/ProjectLogsManager.test.ts

```diff
@@ -1,9 +1,19 @@
-// @ts-nocheck
-const { ProjectLogsManager } = require('../projects/ProjectLogsManager');
-const { getProjectConfig, ensureProjectExists } = require('../projects');
-const {
-  fetchProjectComponentsMetadata,
-} = require('@hubspot/local-dev-lib/api/projects');
+import { ProjectLogsManager } from '../ProjectLogsManager';
+import { getProjectConfig, ensureProjectExists } from '..';
+import { fetchProjectComponentsMetadata } from '@hubspot/local-dev-lib/api/projects';
+
+const SUBCOMPONENT_TYPES = {
+  APP_ID: 'APP_ID',
+  PACKAGE_LOCK_FILE: 'PACKAGE_LOCK_FILE',
+  CRM_CARD_V2: 'CRM_CARD_V2',
+  CARD_V2: 'CARD_V2',
+  SERVERLESS_PKG: 'SERVERLESS_PKG',
+  SERVERLESS_ROUTE: 'SERVERLESS_ROUTE',
+  SERVERLESS_FUNCTION: 'SERVERLESS_FUNCTION',
+  APP_FUNCTION: 'APP_FUNCTION',
+  AUTOMATION_ACTION: 'AUTOMATION_ACTION',
+  REACT_EXTENSION: 'REACT_EXTENSION',
+} as const;
```

Comment:
You can just grab these from LDL

Reply:
I was getting an error when I attempted that--it didn't look to me like the `enums` folder where this is located had an export assigned in LDL's package.json. Am I missing anything?

Reply:
Oh interesting, I didn't realize those aren't exported. That's fine then!